app/views/wizards/claims/add_mentor_wizard/_mentor_step.html.erb

```diff
@@ -0,0 +1,38 @@
+<% caption = current_user.support_user? ? "#{t(".caption")} #{contextual_text}" : t(".caption") %>
+<% content_for :page_title, current_step.errors.any? ? t(".page_title_with_error", error: current_step.errors.first.message, contextual_text: caption) : t(".page_title", contextual_text: caption) %>
```

Comment:
This is causing some strange behaviour.
Unlike with our service, Claims has the "Caption - School name" format, as the Support side has it's own views. You'll need to consider this in the captions and page titles.
![screencapture-track-and-pay-1169-test-teacherservices-cloud-support-schools-fff7c7ed-e9e4-4c2d-833b-e1867987260a-mentors-new-4141932d-1231-4e93-bde6-4afb9ae075dd-mentor-2024-11-07-16_07_12](https://github.com/user-attachments/assets/3c3a7710-ad0b-4a2e-a149-0867291c9129)